src/lib.rs

```diff
@@ -813,6 +824,20 @@ impl Url {
     /// For cannot-be-a-base URLs, this is an arbitrary string that doesn’t start with '/'.
     /// For other URLs, this starts with a '/' slash
     /// and continues with slash-separated path segments.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use url::Url;
+    ///
+    /// let url = Url::parse("https://example.com/api/versions?page=2").unwrap();
+    ///
+    /// assert_eq!(url.path(), "/api/versions");
```

Comment:
I think this would be more readable at a glance if the asserts were grouped with the previous line:

```rust
use url::Url;

let url = Url::parse("https://example.com/api/versions?page=2").unwrap();
assert_eq!(url.path(), "/api/versions");

let url = Url::parse("https://example.com").unwrap();
assert_eq!(url.path(), "/");
```